### DISCUSSION TITLE
[Solved] "Failed to verify authentication" error when loading running and debugging in VSCode

### DISCUSSION THREAD

**Discussion by azrellie** [FabricMC/fabric-api#2352] [Player support]
Whenever I try to run and debug my mod in VSCode, it crashes and gives me a "Failed to verify authentication" error. I have tried to install "Auth Me" mod in my projects 'mod folder' (as seen here) to fix this but it does not help.
![image](https://user-images.githubusercontent.com/98503938/175136386-2eeed448-0052-4535-8074-28826f00ddb5.png)

Crash logs:
[crash-2022-06-22_14.04.05-client.txt](https://github.com/FabricMC/fabric/files/8961304/crash-2022-06-22_14.04.05-client.txt)

**Comment by warjort** [comment]
The error in the crash report doesn't match what you say.
````
Caused by: java.lang.NoClassDefFoundError: softie/test/mod/Testmod (wrong name: softie/test/mod/TestMod)
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1012)
	at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.defineClassFwd(KnotClassLoader.java:186)
````

You have some mismatch between your classname and its configuration in the entrypoint in fabric.mod.json

Testmod is not the same as
TestMod

**Comment by Juuxel** [comment]
The crash is not from authentication but from an incorrect class name for your entrypoint in the fabric.mod.json:

> Caused by: java.lang.NoClassDefFoundError: softie/test/mod/Testmod (wrong name: softie/test/mod/TestMod)

The authentication error is perfectly normal and happens in all dev environments, it's safe to ignore. (Dev envs don't have a Mojang auth token so they can't authenticate for online mode servers)

**Comment by azrellie** [comment]
@Juuxel @warjort
Oh I see. Changed it and it works now. I appreciate your guys help.
